### ISSUE TITLE
afk message

### ISSUE THREAD

**Issue by seema84** [EssentialsX/Essentials#2123]
EssentialsX version: 2.15.0.16

Sometimes join a players in afk mode.

Chat output on join:
1. player is now afk
2. [+] player join the server

**Comment by mdcfe** [comment]
Please fill out the template as requested.

```
<!--
Don't put anything inside this block, as it won't be included in the issue.

If you have a question about using EssentialsX, it may be more helpful to ask
in our Discord server: https://discord.gg/F7gexAQ

If you have a feature suggestion for EssentialsX, please delete the template.

Otherwise, if you are reporting a bug, read on:
1.  Fill out the template, running the commands either in the console or 
    as a player. Don't simply put "latest" or we will ignore it.

2.  When linking files, do not attach them to the post! Paste them on 
    https://gist.github.com/, then paste a link to them in the relevant parts
    of the template. Avoid using Hastebin or Pastebin, as files are deleted
    after a period of time.

3.  If you are reporting an issue with lag, please include a timings report.

4.  If you are reporting an issue with messages or in-game behaviour, please
    include screenshots detailing the problem.

5.  Include a description and any other details that may be helpful under the
    Details section.

6.  Delete this line and all above lines before posting your issue!       -->

### Information

Full output of `/ess version`:




Server log (upload `logs/latest.log` to [Gist](https://gist.github.com/)):

EssentialsX config (upload `plugins/Essentials/config.yml` to [Gist](https://gist.github.com/)): 

### Details

Describe your issue here.

```

**Comment by seema84** [comment]
https://picload.org/view/dllopgol/afk.png.html

Server version: 1.12.2-R0.1-SNAPSHOT git-Spigot-642f6d2-ab13683 (MC: 1.12.2)
EssentialsX version: 2.15.0.16
PermissionsEx version: 1.23.4
Vault version: 1.5.6-b49
EssentialsXChat version: 2.15.0.16

Thats the only one what has been missing.

**Comment by Remski01** [comment]
please send you config too, you can use pastbin https://pastebin.com/

**Comment by seema84** [comment]
https://pastebin.com/TphvhhKy

**Comment by takatalvi** [comment]
Can confirm, even on the latest experimental build dev-23.73. The AFK timer is not reset when the player disconnects so it's possible to join the server while still in AFK.

**Comment by triagonal** [comment]
This was fixed in #2458.
